keyboards/vitamins_included/vitamins_included.h

```diff
@@ -20,6 +47,6 @@
         KC_##L30, KC_##L31, KC_##L32, KC_##L33, KC_##L34, KC_##L35, KC_##R30, KC_##R31, KC_##R32, KC_##R33, KC_##R34, KC_##R35 \
     )
 
-#define KC_LAYOUT_ortho_4x12 KC_KEYMAP
-
-#endif
+#define KC_LAYOUT_ortho_4x12 LAYOUT_kc
+#define LAYOUT_ortho_4x12 LAYOUT
+#define KEYMAP LAYOUT
```

Comment:
For backwards compatibility, you would need to maintain a `KC_KEYMAP` alias for any potential keymap use, and I dont think the `#define KEYMAP LAYOUT` alias is required.

```suggestion
#define KC_KEYMAP LAYOUT_kc
```